### ISSUE TITLE
expose additional Lynx methods in toolkit metrics

### ISSUE THREAD

**Issue by EvaLok** [me-ventures/microservice-toolkit#5]
stuff like `count`, `decrement` and so might be handy for more accurate metrics

**Comment by MaikelH** [comment]
We should definitely be looking into exposing more options. We should make a list of all the different methods and see what we can implement. I think most of the stuff should be pretty easy to implement.

**Comment by EvaLok** [comment]
yep i think so too; we could also consider adding a "native" (or something like this) export on the metrics so that library-specific-but-not-universal methods can be accessed. something like:
```js
var metrics = require('microservice-toolkit').metrics.native;

metrics.someRandomMethod('stuff');
```

what do you think?

**Comment by EvaLok** [comment]
..of course i guess this only applies once we want to use other libraries for metrics

**Comment by MaikelH** [comment]
Exposing the internal metrics could be dangerous, since it can easily break. The underlying library can be changed or a different metrics back-end is loaded and all your code breaks.

**Comment by EvaLok** [comment]
that certainly could happen; i was more thinking that it might be a nice way to stay as unopinionated as possible, since presumably if someone wants access to the native methods, they would be locking in the toolkit version and aware enough of their configuration situation that this wouldn't be a problem. in any case, even if we were to decide to do it, it's definitely not high priority compared to simply exposing some common methods, just wanted to consider it ;)
